src/app/Processor/ProcessorDetailPage/ProcessorDetailPage.tsx

```diff
@@ -215,6 +219,14 @@ const ProcessorDetailPage = (): JSX.Element => {
         actionModalMessage.current = t(
           "processor.errors.cantUpdateProcessorBecauseNotReadyState"
         );
+      } else if (
+        isServiceApiError(updateProcessorError) &&
+        getErrorCode(updateProcessorError) === APIErrorCodes.ERROR_22
+      ) {
+        setMalformedTemplate(
+          getErrorReason(updateProcessorError) ??
```

Comment:
I do not know how display malformedTemplate if I am not able to create a processor with malformed Template.

Reply:
@vpellegrino now I can see that the transformation is editable in `ProcessorDetailPage`. So I confirm that it works here as expected.